docs/Materials.md.html

```diff
@@ -1093,6 +1097,53 @@
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+### Blending and transparency: refraction
+
+Type
+:    `string`
+
+Value
+:     Any of `none`, `cubemap`, `screenspace`. Defaults to `none`.
+
+Description
+:     Activates refraction when set to anything but `none`. A value of `cubemap` will only use the
+      IBL cubemap as source of refraction, while this is significantly more efficient, no scene
+      objects will be refracted, only the distant environment encoded in the cubemap. This mode is
+      adequate for an object viewer for instance. A value of `screenspace` will employ the more
+      advanced screen-space refraction algorithm which allows opaque objects in the scene to be
+      refracted. In `cubemap` mode, refracted rays are assumed to emerge from the center of the
+      object. In `screenspace` mode, refracted rays are assumed to travel parallel to the view
+      direction when they exit the refractive medium.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ JSON
```

Comment:
TODO: we should add screenshots for those 2 properties when things are working.

Reply:
yes